packages/amplify-console-integration-tests/src/consoleHosting/utils.ts

```diff
@@ -82,7 +82,7 @@ export async function createTestProject(): Promise<string> {
   const projectName = path.basename(projRoot);
   const projectDir = path.dirname(projRoot);
 
-  spawnSync('npx', ['create-react-app', '--scripts-version', '4.0.3', projectName], { cwd: projectDir });
+  spawnSync('npx', ['create-react-app', '--scripts-version', '5.0.1', projectName], { cwd: projectDir });
```

Comment:
This update is related to https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported .
I.e. react scripts 4.x are outdated and bring webpack that relies on older ssl.